src/styles/app/42-calendar.css

```
@@ -55,53 +42,128 @@
+
+.calendarAccentButton:hover,
+.calendarAccentButton:focus-visible,
+.calendarAccentIconButton:hover,
+.calendarAccentIconButton:focus-visible {
+	background: var(--interactive-accent-hover);
+	color: var(--text-inverse);
+}
+
+.calendarAccentButton svg,
+.calendarAccentIconButton svg {
+	color: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)